README.md

```diff
@@ -161,13 +161,15 @@ Of course you could proxy all requests to `ws.example.com` to port 8080 of your
 
 # Configuration Options
 
-Slanger supports several configuration options, which can be supplied as command line arguments at invocation.
+Slanger supports several configuration options, which can be supplied as command line arguments at invocation. You can also supply a yaml file containing config options. If you use the config file in combination with other configuration options, the values passed on the command line will win. Allows running multiple instances with only a few differences easy.
 
 ```
 -k or --app_key This is the Pusher app key you want to use. This is a required argument
 
 -s or --secret This is your Pusher secret. This is a required argument
 
+-C or --config_file Path to Yaml file that contains any desired configuration options
```

Comment:
Should be easy to clean up. I was trying to avoid touching other lines. Open to suggestions, but my first thought is to modify the -k and -s lines to say either "required option on command line or via config file". Didn't want to create a description that was too long. The same validations are in place, so if they don't have the values set, service won't start. 
As for use case - nothing specific other than when setting up the service to run I didn't like having to set all those options via the command line and also show up in the history. I also wanted the ability to just hand someone a config file they can run for testing, so yes, something like chef or one of the other configuration management tools.